crates/platform/resources/storage/surrealdb/pvc.yaml

```
@@ -0,0 +1,15 @@
+  name: {{ name }}
+  namespace: {{ namespace }}
+  labels:
+    app.kubernetes.io/name: {{ name }}
+    app.kubernetes.io/component: database
+    app.kubernetes.io/part-of: {{ part.of }}
+    app.kubernetes.io/version: {{ version }}
+spec:
+  accessModes: [ "ReadWriteOnce" ]
+  resources:
+    requests:
+      storage: {{ storage.quantity }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)